docs/src/routes.tsx

```diff
@@ -94,6 +98,10 @@ const Router = () => (
           />,
         ]}
         <DocsLayout exact path="/accessibility" component={Accessibility} />
+        <DocsLayout exact path="/accessibility-behaviors" component={AccessibilityBehaviors} />
```

Comment:
don't see the code that introduces these docs to the sidebar menu, its Guides section - is it missed?

Reply:
it wouldn't be visible. All navigation to accessibility-related docs is introduced on the main accessibility page.

Reply:
![image](https://user-images.githubusercontent.com/8460706/55724022-d0c11b80-5a0a-11e9-8a52-fc7934ef6a74.png)
![image](https://user-images.githubusercontent.com/8460706/55724165-1ed61f00-5a0b-11e9-9069-47a11996ff37.png)